lib/client.js

```diff
@@ -179,6 +179,12 @@ var warmupLimit = 32;
  *   determine if an statement can be retried in case of request error or write timeout.
  * </p>
  * <p>Default: <code>false</code>.</p>
+ * @property {String} [keyspace] Specifies the keyspace for the query. Used for routing within the driver, this
```

Comment:
Ah I stand corrected, in the java driver there is a way to set the keyspace on a simple statement for this exact case so I was incorrect about that.